vmdb/app/controllers/application_controller.rb

```diff
@@ -966,8 +966,8 @@ def get_reports_menu(group=session[:group], tree_type="reports", mode="menu")
     rec = MiqGroup.find_by_id(group)
     user = current_user
     @sb[:grp_title] = user.admin_user? ?
-      "#{session[:customer_name]} (#{_("All %s") % ui_lookup(:models=>"MiqGroup")})" :
-      "#{session[:customer_name]} (#{_("%s") % "#{ui_lookup(:model=>"MiqGroup")}: #{user.current_group.description}"})"
+      "#{current_brand.customer_name} (#{_("All %s") % ui_lookup(:models=>"MiqGroup")})" :
+      "#{current_brand.customer_name} (#{_("%s") % "#{ui_lookup(:model=>"MiqGroup")}: #{user.current_group.description}"})"
```

Comment:
Not sure what happens when you get a sub-tenant. I was thinking that multiple tenants may point to the same brand object in the database.
It seemed like @epwinchell had quite a number of additional attributes that could go into brand like colors and hrefs.